subsys/mgmt/osdp/Kconfig.cp

```diff
@@ -41,3 +41,14 @@ config OSDP_PD_POLL_RATE
 	  The Control Panel must query the Peripheral Device periodically to
 	  maintain connection sequence and to get status and events. This option
 	  defined the number of times such a POLL command is sent per second.
+
+if OSDP_SC_ENABLED
+
+config OSDP_MASTER_KEY
+	string "Secure Channel Master Key"
+	default "00112233445566778899aabbccddeeff"
```

Comment:
This forces all the devices, that use this zephyr image, to have same master key. Is this really intended to work like this?

Reply:
This key must be unique to each device. This was a arbitrary default I added. Adding this to prj.conf also doesn't make a difference here. Perhaps this should come from devicetree?

Reply:
For now, I've moved it into the prj.conf where its more visible that each app must set this.

Doing this causes a harmless warning when OSDP mode is set to PD. I'll split the samples (later today) into CP and PD and it will go away.
```
warning: OSDP_MASTER_KEY (defined at subsys/mgmt/osdp/Kconfig.cp:47) was assigned the value
'11223344556677889911bbccddeeff' but got the value ''. Check these unsatisfied dependencies:
OSDP_MODE_CP (=n).
```